lib/node_modules/@stdlib/lapack/base/dlange/lib/norms.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
No need for an object. Just include the array in this file. Having a nested object just waste characters and space.